GetDKAN/dkan
# Add bootstrap_library module

connects #2667 
- Using Bootstrap JS from CDN causes cross-domain javascript source file inclusion security errors.
- Using bootstrap_library breaks recline and dropdown menu functionality.
- Patching the radix theme to use a local bootstrap.min.js file.

## QA Steps
- View page source, confirm you do not see https://maxcdn.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js
- Test the Topics menu, confirm the dropdown works
- Visit resource node with csv preview, confirm recline js works
